hector_mapping/src/HectorMappingRos.h

```diff
@@ -115,6 +120,9 @@ class HectorMappingRos
   ros::Publisher odometryPublisher_;
   ros::Publisher scan_point_cloud_publisher_;
 
+  ros::ServiceServer resetMapService_;
+  ros::ServiceServer toggleScanProcessingService_;
```

Comment:
ros service servers

Reply:
Usually, we use snake_case for variable names, though, this is a bit inconsistent in this repo.

Reply:
sounds good! that goes along with [Google's Style Guide](https://google.github.io/styleguide/cppguide.html#Variable_Names)! I will make these variables snake_case, and maybe I'll spend sometime in the future to fix this whole class (in another PR).